src/main/java/org/openrewrite/java/testing/jmockit/JMockitBlockRewriter.java

```diff
@@ -242,9 +251,13 @@ private J.Block rewriteTemplate(String verifyTemplate, List<Object> templatePara
                 );
     }
 
-    private static @Nullable String getWhenTemplate(List<Expression> results) {
+    private @Nullable String getWhenTemplate(List<Expression> results) {
         boolean buildingResults = false;
-        final StringBuilder templateBuilder = new StringBuilder(WHEN_TEMPLATE_PREFIX);
+        StringBuilder templateBuilder = new StringBuilder();
+        if (this.blockType == NonStrictExpectations) {
+            templateBuilder.append("lenient().");
```

Comment:
You need to add the import for lenient as well here.

Reply:
Don't like adding constants if not needed in multiple places, or needed to easily modify, prefer keeping things in context within the context. What is reason for static final String?

Reply:
It's already a convention in this recipe to use static final String for template strings, so let's stay consistent.

It also makes it easier to identify what templates we are using, rather than forcing users to read through all the code.